CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Changelog
 Running changelog of releases since `1.0.0-beta01`
 
-## v1.0.0-beta03
+## v1.0.0
```

Comment:
Skipped beta03 to focus on implementing all of v1

Reply:
I suggest adding a short description indicating that this is the first stable version 🎉